pkg/netns/netns_windows.go

```diff
@@ -18,33 +18,57 @@ limitations under the License.
 
 package netns
 
-// TODO(windows): Implement netns for windows.
-// NetNS holds network namespace.
+import "github.com/Microsoft/hcsshim/hcn"
+
+// NetNS holds network namespace for sandbox
 type NetNS struct {
+	path string
 }
 
-// NewNetNS creates a network namespace.
+// NewNetNS creates a network namespace for the sandbox
 func NewNetNS() (*NetNS, error) {
-	return nil, nil
+	temp := hcn.HostComputeNamespace{}
+	hcnNamespace, err := temp.Create()
+	if err != nil {
+		return nil, err
+	}
+
+	return &NetNS{path: string(hcnNamespace.Id)}, nil
 }
 
 // LoadNetNS loads existing network namespace.
 func LoadNetNS(path string) *NetNS {
-	return nil
+	return &NetNS{path: path}
 }
 
-// Remove removes network namepace. Remove is idempotent, meaning it might
-// be invoked multiple times and provides consistent result.
+// Remove removes network namepace if it exists and not closed. Remove is idempotent,
+// meaning it might be invoked multiple times and provides consistent result.
 func (n *NetNS) Remove() error {
-	return nil
+	hcnNamespace, err := hcn.GetNamespaceByID(n.path)
+	if err == nil {
+		return hcnNamespace.Delete()
```

Comment:
I suppose its the same race here as well right? 

Should this read:
```go
hcnNamespace, err := hcn.GetNamespaceByID(n.path)
if err == nil {
    err = hcnNamespace.Delete()
}
if hcn.IsNotFound(err) {
    return nil
}
return err
```

Reply:
Hm, if `Delete` behaves like this, we can just call `Delete` and check error.

Let me try.

Reply:
Done